integrations/stripe/subscriptions/server/node/src/diagonal.ts

```diff
@@ -145,12 +149,13 @@ export async function handleDiagonalRequest(
                     },
                 )
 
-                // [Optional]: Manually force Stripe to send invoice to the customer
-                // attached to the invoice. This is not necessary, as Stripe will send
-                // the invoice automatically once it's marked as paid.
-                await stripe.invoices.sendInvoice(charge.reference, {
-                    idempotencyKey: 'send-' + charge.id,
-                })
+                await EmailClient.sendInvoicePaymentSuccessWallet(
```

Comment:
Should this be in a try-catch or smth?

Reply:
Thought about it, but then adds verbosity when reading the code - and at this point isn't strictly required